.travis.yml

```diff
@@ -15,14 +15,14 @@ matrix:
       rust: beta
       env: GTK=3.24 FEATURES=subclassing,v2_48
     - os: linux
-      rust: 1.34.0
+      rust: 1.36.0
       env: GTK=3.14 FEATURES=subclassing
     - os: linux
-      rust: 1.34.0
+      rust: 1.36.0
       env: GTK=3.24 FEATURES=subclassing,v2_48
     - os: osx
       rust: nightly
-      env: GTK=3.14 FEATURES=subclassing,futures
+      env: GTK=3.16 FEATURES=subclassing,futures
```

Comment:
Why? 3.14 is still the minimum version we support or not?

Reply:
Doesn't `MaybeUninitialized` gets stabilized in 1.36?

Reply:
Yes but here you change the version for GTK, not Rust :)

Reply:
Oh damn, good catch!